src/components/Tank/Tank.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+import { getBgPosition, getTankPositions } from './Tank.helpers';
+import { TankProps } from './Tank.model';
+import styles from './Tank.module.scss';
+
+const propsDefault: Readonly<Required<TankProps>> = {
+  color: 'yellow',
+  position: 'up',
+  type: '0'
+};
+
+// eslint-disable-next-line complexity
```

Comment:
- [ ] Can you please figure out why where your component's complexity hat big?

Reply:
In our linter, we have `"complexity": ["error", 3],` 
Check out [this](https://eslint.org/docs/rules/complexity). We also got an error when we have more than 3 cases(counted `default` as well) in switch-block :man_facepalming: [link](https://github.com/kiraki-dev/kiraki-games/pull/14/files#diff-33390961c5a1947a011ceb042b9ef451741275f94a3d7686141982d7d6b049bdR61)
I suggest changing this somehow to not affect switches.

Reply:
string to function maps/objects instead.